Documentation/metrics.md

```diff
@@ -30,6 +30,37 @@ Pending proposal (`pending_proposal_total`) gives you an idea about how many pro
 
 Failed proposals (`proposal_failed_total`) are normally related to two issues: temporary failures related to a leader election or longer duration downtime caused by a loss of quorum in the cluster.
 
+
+### store
+
+These metrics describe the accesses into the data store of Etcd nodes that exist in the cluster (non-proxy). They 
+are useful to count what kind of actions are taken by users, and whether all Etcd nodes "see" the same set of data.
+
+All these metrics are prefixed with `etcd_store_`
+
+| Name                      | Description                                                                          | Type                   |
+|---------------------------|------------------------------------------------------------------------------------------|--------------------|
+| reads_total               | Reads from store, should differ across all Etcd nodes (local reads).                 | Counter(action,outcome)|
+| writes_total              | Writes to store, should be similar across all Etcd nodes.                            | Counter(action,outcome)|
+| expires_total             | Count of expired keys (due to TTL).                                                  | Counter                |
+| watch_requests_totals     | Watch requests received by this node.                                                | Counter                | 
+| watchers                  | Current count of active watchers on the given node.                                  | Gauge                  |
+
+Example Prometheus queries that may be useful from these metrics (across all Etcd servers):
+
+ *  `sum(rate(etcd_store_reads_total{job="etcd"}[1m])) by (action)`
+    `max(rate(etcd_store_writes_total{job="etcd"}[1m])) by (action)`
+    
+    Rate of reads and writes by action, across all servers across a time window of `1m`. The reason why `max` is used for writes as opposed to `sum` for reads is because all of Etcd nodes in the cluster apply all writes to their stores.
```

Comment:
across all etcd members... there are a few other places need to be changed. i will not repeat myself. :)